superset-frontend/src/dashboard/components/SaveModal.tsx

```diff
@@ -137,17 +152,17 @@ class SaveModal extends React.PureComponent {
         t('You must pick a name for the new dashboard'),
       );
     } else {
-      this.onSave(data, dashboardId, saveType).then(resp => {
+      this.onSave(data, dashboardId, saveType).then((resp: any) => {
         if (
           saveType === SAVE_TYPE_NEWDASHBOARD &&
           resp &&
           resp.json &&
           resp.json.id
         ) {
-          window.location = `/superset/dashboard/${resp.json.id}/`;
+          window.location.href = `/superset/dashboard/${resp.json.id}/`;
```

Comment:
oo, was this a bug that typescript caught?

Reply:
Setting `location` is equivalent to setting `location.href`.

Reply:
Wow, you calling me a youngin?!? 😆

Reply:
Well, _back when I_ started building websites, there wasn't even JavaScript yet! Or CSS! Gather round, let me spin you a yarn about the old days. You see, there was this browser called Mosaic...

... hey, are you kids even listening?